### ISSUE TITLE
dapp create --verify fails with seth --to-hexdata

### ISSUE THREAD

**Issue by realisation** [dapphub/dapptools#447]
```
Verifying contract at 0x8dc2a50faea58cb3732eae0803dd08364d8b44d6
+ [[ -n F68P23IXYDN4KSPBSXSP97ZCAHVWC1K9KZ ]]
++ seth chain
+ chain=kovan
+ case "$chain" in
+ export ETHERSCAN_API_URL=https://api-kovan.etherscan.io/api
+ ETHERSCAN_API_URL=https://api-kovan.etherscan.io/api
+ export ETHERSCAN_URL=https://kovan.etherscan.io/address
+ ETHERSCAN_URL=https://kovan.etherscan.io/address
+ path=src/assimilators/kovan/susdReserves/kovanSUsdToSUsdAssimilator.sol:KovanSUsdToSUsdAssimilator
+ name=KovanSUsdToSUsdAssimilator
+ address=0x8dc2a50faea58cb3732eae0803dd08364d8b44d6
++ seth calldata ''
seth---to-hexdata: error: invalid hexdata: `'
+ constructorArguments=
0x8dC2a50faeA58cB3732EAE0803Dd08364d8B44d6
```

**Comment by realisation** [comment]
#448

**Comment by MrChico** [comment]
Closed by #448
